cmd/hegelc/main.go

```diff
@@ -4,36 +4,65 @@ import (
 	"context"
 	"crypto/tls"
 	"errors"
+	"flag"
 	"fmt"
 	"io"
 	"log"
+	"os"
+	"strconv"
 
 	"github.com/tinkerbell/hegel/grpc/protos/hegel"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
+var (
+	server string
+	port   int
+
+	defaultServer = "metadata"
+	defaultPort   = 50060
+
+	envVarServer = "HEGEL_SERVER"
+	envVarPort   = "HEGEL_PORT"
+)
+
 func main() {
+	if envServer := os.Getenv(envVarServer); envServer != "" {
+		defaultServer = envServer
+	}
+	if envPort := os.Getenv(envVarPort); envPort != "" {
+		var err error
+		if defaultPort, err = strconv.Atoi(envPort); err != nil {
+			log.Panic(err)
```

Comment:
fyi: github.com/packethost/pkg/env has a bunch of nice funcs that do these conversions, you might want to use that, but not going to require it.